dashboard-app/src/app/pages/hotel-info/controllers/HotelCtrl.js

```diff
@@ -0,0 +1,31 @@
+(function () {
+    'use strict';
+
+    angular.module('ConciergeApp.pages.hotelInfo')
+        .controller('HotelCtrl', HotelCtrl);
+
+    HotelCtrl.$inject = ['HotelService', '$scope'];
+    function HotelCtrl(HotelService, $scope) {
+        $scope.hotel = {};
+
+        // getting countries from database to show in select box
+        HotelService.countries().then(function (response) {
+            $scope.countries = response.data;
+        }, function (errorMessage) {
+            console.log(errorMessage);
+        });
+
+        HotelService.getHotel().then(function(response){
```

Comment:
Reformat code please. Add space between statements and curly braces. Also, do you put space after `function` keyword or not? Stick to unique coding style.